local_store.go

```diff
@@ -43,6 +42,8 @@ type LocalStore interface {
 	Commit(bool, map[string]int, map[string]data.Hashes) error
 
 	// GetSigners return a list of signers for a role.
+	// This may include revoked keys, so the signers should not
+	// be used without filtering.
```

Comment:
> The consequence of failing to filter would be having signatures that don't verify, correct

Correct. 

> It's not a security issue, just an availability one.

I don't think it's a security issue -- more of a usability issue. Signing with untrusted or revoked keys does not present any security risk in TUF, since keys must be explicitly trusted in the client workflow before the signatures count toward a threshold.

> Is the information in LocalStore sufficient to do this filtering

You need a `verify.DB` or something equivalent to do the filtering, which is outside the scope of what `GetSigners` does.

Reply:
Also note that the root role needs to sign with expired keys sometimes, so there's no universal way to verify whether a key is valid without a lot of context.